src/program/canvas/screens/loading/loading.screen.ts

```diff
@@ -17,13 +17,14 @@ export class LoadingScreen extends ScreenAbstract {
             fontSize: 8,
             fill: '#9bbc0f'
         }));
+        this.textGUI.position.set(-10, 0)
```

Comment:
Can we add a comment explaining why this initial position?  (avoiding magic numbers)

Reply:
This is only aprt of the example, not big deal.